resources/docs/troubleshooting-parallel.md

```diff
@@ -32,6 +31,11 @@ public function parallel(
 ): void
 ```
 
+To manually set these options, call the `parallel` function within the Rector config class:
+```php
+$rectorConfig->parallel(120, 16, 20);
```

Comment:
Rector 0.19.0 is using `60, 32, 20` as default value, ref

https://github.com/rectorphp/rector-src/blob/ac072bc8af186b2c86256aa7d29e52bb608ece27/src/Config/RectorConfig.php#L91

Could you update the default value on line 28 to 30 above it?

Reply:
Done :)